src/Nox.Solution/Models/Domain/Entities/Entity.cs

```diff
@@ -81,6 +81,11 @@ public class Entity : YamlConfigNode<NoxSolution, Domain>
     [AdditionalProperties(false)]
     public IReadOnlyList<UniqueAttributeConstraint> UniqueAttributeConstraints { get; internal set; } = Array.Empty<UniqueAttributeConstraint>();
 
+    [Title("Depth of relationship navigation.")]
+    [Description("Specifies the depth for generating related entities endpoints.")]
+    [Minimum(1), Maximum(5)]
+    public int ManageRelationshipDepth { get; internal set; } = 1;
```

Comment:
```suggestion
    public int ApiRelationshipMaxDepth { get; internal set; } = 1;
```

Reply:
We need to think to move this to the API level with the CanManageEntity and CanManageRef